vms/platformvm/block/executor/options.go

```diff
@@ -68,15 +93,15 @@ func (o *options) ApricotProposalBlock(b *block.ApricotProposalBlock) error {
 	nextHeight := b.Height() + 1
 
 	var err error
-	o.commitBlock, err = block.NewApricotCommitBlock(blkID, nextHeight)
+	o.preferredBlock, err = block.NewApricotCommitBlock(blkID, nextHeight)
 	if err != nil {
 		return fmt.Errorf(
 			"failed to create commit block: %w",
 			err,
 		)
 	}
 
-	o.abortBlock, err = block.NewApricotAbortBlock(blkID, nextHeight)
+	o.alternateBlock, err = block.NewApricotAbortBlock(blkID, nextHeight)
```

Comment:
We don't really care about the preferences for `Apricot` blocks at this point, because they have all been accepted already. We could probably change some code to actually remove this logic... But it's out of scope for this PR.